src/scripts/choices.js

```diff
@@ -569,7 +569,9 @@ class Choices {
               console.error(err);
             }
           })
-          .then(() => this._handleLoadingState(false))
+          .then(() =>
+            requestAnimationFrame(() => this._handleLoadingState(false)),
+          )
```

Comment:
It was wrapped in `requestAnimationFrame` in original `.ajax` version. @jshjohnson do you remember why?